Python/web_scraping_Amazon

```diff
@@ -0,0 +1,76 @@
+#Amazon web scraping
+#website selected = Amazon
+
+#Scraps a page from Amazon website and collects all the product related information from the website and store them in a data frame.
+
+import pandas as pd
+import numpy as np
+import re
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+import requests
+
+
+no_pages = 1
+
+def get_data(pageNo):  
+    
+    r = requests.get('https://www.amazon.in/gp/bestsellers/dvd/21360334031/ref=zg_bs_pg_?'+str(pageNo)+'ie=UTF8&pg='+str(pageNo))
```

Comment:
You can use 'f strings' to keep your code clean like:
f'https://www.amazon.in/gp/bestsellers/dvd/21360334031/ref=zg_bs_pg_?{str(pageNo)}ie=UTF8&pg={str(pageNo)}'